docker-for-mac/release-notes.md

```diff
@@ -13,13 +13,43 @@ This page contains information about the new features, improvements, known issue
 
 > **Important**
 >
-> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
+> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background. When the download finishes, all you need to do is to click **Update and restart** to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Upgrades
+
+- [Docker Compose 1.29.0](https://github.com/docker/compose/releases/tag/1.29.0)
+- [Compose CLI v1.0.12](https://github.com/docker/compose-cli/tree/v1.0.12)
+- [Linux kernel 5.10.25](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.25-6594e668feec68f102a58011bb42bd5dc07a7a9b/images/sha256-80e22cd9c9e6a188a785d0e23b4cefae76595abe1e4a535449627c2794b10871?context=repo)
+- [Snyk v1.461.0](https://github.com/snyk/snyk/releases/tag/v1.461.0)
+- [Docker Hub Tool v0.3.1](https://github.com/docker/hub-tool/releases/tag/v0.3.1)
+- [containerd v1.4.4](https://github.com/containerd/containerd/releases/tag/v1.4.4)
+- [runc v1.0.0-rc93](https://github.com/opencontainers/runc/releases/tag/v1.0.0-rc93)
+
+### Bug fixes and minor changes
+
+- Fixed an issue related to docker-compose action. Fixes [docker/for-win#10564](https://github.com/docker/for-win/issues/10564).
+- Fixed launching Docker Desktop from the Dock. Fixes [docker/for-mac#5515](https://github.com/docker/for-mac/issues/5515).
+- Ensure `--add-host host.docker.internal:host-gateway` causes `host.docker.internal` resolves to the host IP, rather than the IP of the IP router. See [docker/for-linux#264](https://github.com/docker/for-linux/issues/264).
```

Comment:
```suggestion
- Fixed a bug that `--add-host host.docker.internal:host-gateway` caused `host.docker.internal` to resolve to the wrong IP address.
```

Also the ticket cited is not really relevant: is there a specific comment within the ticket we could link to? If not, just omit it.

Reply:
I’m not sure, @djs55 any recollection of this? It’s from https://github.com/docker/pinata/pull/15504

Reply:
It should fix the issue where the new `host-gateway` feature in the engine fixes Linux but breaks macOS: https://github.com/docker/for-linux/issues/264#issuecomment-785137844